etcdserver/api/v3rpc/key.go

```diff
@@ -39,6 +41,7 @@ type kvServer struct {
 }
 
 func NewKVServer(s *etcdserver.EtcdServer) pb.KVServer {
+	MaxOpsPerTxn = int(s.Cfg.MaxOpsPerTxn)
```

Comment:
@heyitsanthony  I am thinking about each KvServer should hold its own MaxOpsPerTxn stored in
```go
type kvServer struct {
	hdr header
	kv  etcdserver.RaftKV
        maxTxnOps unint
}
```

and use kvServer.maxTxnOps for transaction ops check?